### PR TITLE
Update library

### PR DESCRIPTION
**Pull request by giovannicuriel**
This PR updates dependencies versions. It also updates the documentation.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -22,22 +23,23 @@ class IoTAgent {
   constructor() {
 
     // list of implementation required callbacks.
-    this.messenger = new dojotModule.Messenger("iotagent");
+    dojotConfig.kafka.consumer["group.id"] = process.env.KAFKA_GROUP_ID || "ioagent";
```
Comment (ericbaum) [inline lib/index.js]: Is the default value correct? ioagent?

Comment (ericbaum) [inline lib/index.js]: If I recall correctly the key "group.id" changed to group_id somewhere else

Comment (giovannicuriel) [reply lib/index.js]: [It has a dot](https://github.com/Blizzard/node-rdkafka#kafkakafkaconsumer).

Comment (ericbaum) [reply lib/index.js]: ok

**Pull request merged**